pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -135,6 +135,21 @@ type PrefilterPlugin interface {
 	Prefilter(pc *PluginContext, p *v1.Pod) *Status
 }
 
+// FilterPlugin is an interface for Filter plugins. These plugins are called at the
+// filter extension point for filtering out hosts that cannot run a pod.
+// This concept used to be called 'predicate' in the original scheduler.
+// These plugins should return only Success or Error in Status.code. However,
+// the scheduler accepts other valid codes as well. Anything other than Success
+// will lead to exclude hosts from running the pod.
+type FilterPlugin interface {
+	Plugin
+	// Filter is called by the scheduling framework.
+	// All FilterPlugins should return sucecss to declare that the node fits the pod.
+	// If Filter returns error, please refer scheduler/algorithm/predicates/error.go
```

Comment:
error -> unschedulable

Reply:
It has been addressed in the latest code.